examples/ex3p.cpp

```diff
@@ -159,9 +167,9 @@ int main(int argc, char *argv[])
    //    by marking all the boundary attributes from the mesh as essential
    //    (Dirichlet) and converting them to a list of true dofs.
    Array<int> ess_tdof_list;
+   Array<int> ess_bdr(pmesh->bdr_attributes.Max());
```

Comment:
I think this can cause an issue if a processor does not own any bdr element, i.e., the array on the processor is empty. So it's better to do 

````cpp
   Array<int> ess_tdof_list;
   Array<int> ess_bdr;
   if (pmesh->bdr_attributes.Size())
   {
      ess_bdr.SetSize(pmesh->bdr_attributes.Max());
      ess_bdr = 1;
 

````

Reply:
Nice catch, thanks.